build.sbt

```diff
@@ -948,6 +948,9 @@ lazy val testUtils = (project in utils("test-utils")).
         "ch.qos.logback" % "logback-classic" % logbackV,
         "commons-io" % "commons-io" % flinkCommonsIOV,
         "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionsCompatV,
+        "com.softwaremill.sttp.client3" %% "slf4j-backend" % sttpV,
+        "org.typelevel" %% "cats-effect" % "2.5.5",
```

Comment:
Let's extract the version to a separate variable. BTW @lciolecki opened this PR: https://github.com/TouK/nussknacker/pull/4287